Packs/Exabeam/Integrations/Exabeam/Exabeam.yml

```diff
@@ -84,6 +84,12 @@ configuration:
   name: first_fetch
   type: 0
   required: false
+- display: 'Advanced: Minutes to look back when fetching'
+  name: look_back
+  type: 0
+  required: false
+  defaultvalue: 1
+  additionalinfo: Use this parameter to determine how long backward to look in the search for incidents that were created before the last run time and did not match the query when they were created.
```

Comment:
Add also the limitation if there is one (max lookback)

Reply:
The API documentation did not contain any limitations